source/features/mark-private-orgs.tsx

```diff
@@ -22,9 +22,9 @@ async function init(): Promise<false | void> {
 		return false;
 	}
 
-	const publicOrganizations = await getPublicOrganizations(getUsername());
+	const publicOrganizationsNames = await getPublicOrganizationsNames(getUsername());
 	for (const org of orgs) {
-		if (!publicOrganizations.includes(org.pathname.slice(1))) {
+		if (!publicOrganizationsNames.includes(org.pathname.replace(/^\/(organizations\/)?|\/$/, ''))) {
```

Comment:
I think this is equivalent
```suggestion
		if (!publicOrganizationsNames.includes(org.pathname.replace(/^\/(organizations\/)?$/, ''))) {
```

Reply:
I thought so as well when I first read it (`/` followed by optional `organizations/`), but:

```
> '/organizations/getferdi'.replace(/^\/(organizations\/)?$/, '')
'/organizations/getferdi'
> '/getferdi'.replace(/^\/(organizations\/)?$/, '')
'/getferdi'
```

Then I noticed I mistakenly added the `$`:

```
> '/organizations/getferdi'.replace(/\/(organizations\/)?/, '')
'getferdi'
> '/getferdi'.replace(/\/(organizations\/)?/, '')
'getferdi'
```

Reply:
Addressed via bf1d679. Thanks for bearing with me! 😄